embarklabs/blog
# Update the atom.xml format

**Updates**

- Add `author` to  atom.xml. For example,
```
<author>
        <name>eric_mastro</name>
</author>
```

- Use text for "summary" which was an html before.
```
from
<summary type="html"> <p><img src="/assets/images/mythx_dashboard.png" alt="Embark dashboard with MythX"></p> <h2 id="How-MythX-works"><a href="#How-MythX-works" </summary>

to 
<summary>Analyse Smart Contract security throughout the development lifecycle using the Embark MythX plugin.</summary>
```

**XML format examples**

- Before: https://blog.embarklabs.io/atom.xml

- After: https://drive.google.com/file/d/1YC3Ts_AhBzCYhACIEQn5IUxBRNvzcn5N/view?usp=sharing

The summary and author information will be used at https://tutorials.statusnetwork.com/ (like Nimbus)

**Test**

1. Run `hexo generate`

2. Check out `public/atom.xml`
